app.js

```diff
@@ -42,6 +44,8 @@ app.use(requestId());
 app.use(logger('dev'));
 app.use(configMiddleware);
 app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerDoc));
+app.use(bodyParser.json());
```

Comment:
This _should_ be ok. The `bodyParser` was after the `/harvest` endpoint as that endpoint was dealing with raw data. That has since changed but that change itself needs to be validated with others from a design point of view.

Reply:
Ok, I'll leave as is for now, we can change later if needed.